### DISCUSSION TITLE
Support for JetBrains IDE databases

### DISCUSSION THREAD

**Discussion by TheLiamGuy** [Infisical/infisical#5633] [Feature Request]
### Feature description
Several JetBrains IDEs (e.g. Pycharm, Datagrip) have strong database tooling built in. Currently the credentials appear to be hardcoded. It would be great if we could somehow use Infisical secrets directly in the config

### Why would it be useful?
Supports secret use within popular JetBrains IDEs

**Comment by afrieirham** [comment]
@TheLiamGuy is this issue connected with #646?

**Comment by TheLiamGuy** [comment]
@afrieirham I don't use TeamCity, but I don't think so; I'd expect the way credentials are managed in cloud/server based CI Pipeline software to be different to a downloaded IDE, even if both are software from the same company
